waterlog/src/Components/AlertTable.js

```diff
@@ -37,22 +37,29 @@ class AlertTableComponent extends Component {
 		return (
 			<table>
 				<thead>
-					<tr>{this.alertInfo.map((info) => <th key = {info}>{info}</th>)}</tr>
+					<tr>
+						{this.alertInfo.map(info => (
+							<th key={info}>{info}</th>
+						))}
+					</tr>
 				</thead>
 				<tbody>
-					{leaks.map((alert) => (
+					{leaks.map(alert => (
 						<tr
-							key = {alert.id}
-							className = {alert.status === 'Fault' ? 'fault' : ''}
-							onClick = {() => (window.location.href = `alert/segment/${alert.id}`)}
+							key={alert.id}
```

Comment:
Why have you condensed the code? It was correctly formatted before. Keep it nice and consistent with appropriate spaces.

Reply:
sure thanks, will consider it next time.

Reply:
Definitely implement it next time :)